ollama/_client.py

```diff
@@ -48,16 +86,15 @@ def __init__(
     `kwargs` are passed to the httpx client.
     """
 
-    headers = kwargs.pop('headers', {})
-    headers['Content-Type'] = 'application/json'
-    headers['Accept'] = 'application/json'
-    headers['User-Agent'] = f'ollama-python/{__version__} ({platform.machine()} {platform.system().lower()}) Python/{platform.python_version()}'
-
     self._client = client(
       base_url=_parse_host(host or os.getenv('OLLAMA_HOST')),
       follow_redirects=follow_redirects,
       timeout=timeout,
-      headers=headers,
+      headers={
+        'Content-Type': 'application/json',
+        'Accept': 'application/json',
+        'User-Agent': f'ollama-python/{__version__} ({platform.machine()} {platform.system().lower()}) Python/{platform.python_version()}',
+      }.update(headers or {}),
```

Comment:
`.update()` is going to return a `None` which means no headers will get passed in